maintainers/README.md

```diff
@@ -172,3 +172,39 @@ to the team without an approval by at least one existing member.
 Various utility scripts, which are mainly useful for nixpkgs maintainers,
 are available under `./scripts/`.  See its [README](./scripts/README.md)
 for further information.
+
+# Guidelines for Committers
+
+When merging changes, care must be taken to reduce impact to the `master`
+branch. If a commit breaks evaluation in `master` it could make it into new PR
+branches and cause further breakage.
+
+One approach to avoid merging such problematic changes is to wait for
+successful Ofborg evaluation. Additionally, using tools like `nixpkgs-review`
+can help spot issues early, before Ofborg finishes evaluation.
+
+## Breaking changes
+
+In general breaking changes to `master` and `staging` branches are permitted.
+Though restrictions might apply towards the end of a NixOS release cycle. This
+is to avoid large-scale breakages shortly before or during a Zero Hydra
+Failures (ZHF) campaign. These restrictions also intend to decrease the
+likelihood of a delayed NixOS release.
```

Comment:
```suggestion
In general breaking changes to `master` and `staging` branches are permitted.
Though restrictions might apply towards the end of a NixOS release cycle, due to
our feature freeze mechanism.
This is to avoid large-scale breakages shortly before and during a Zero Hydra
Failures (ZHF) campaign. These restrictions also intend to decrease the
likelihood of a delayed NixOS release. The feature freeze period is documented
in the announcement of each release schedule.
```